index.html

```diff
@@ -696,6 +751,57 @@
         <li>When the request has been completed, resolve <var>promise</var> with <code>true</code>.
         </li>
       </ol>
+      <p>
+        The serializer for a <a><code>PushSubscription</code></a> invokes the following steps:
+      </p>
+      <ol>
+        <li>Let <var>map</var> be an empty map.
+        </li>
+        <li>Add an entry to <var>map</var> whose key name is <code>endpoint</code> and whose value
+        is the result of converting the <code>endpoint</code> attribute of the
+        <a><code>PushSubscription</code></a> to a serialized value.
+        </li>
+        <li>Let <var>keys</var> be an empty map.
+        </li>
+        <li>For each identifier <var>i</var> corresponding to keys in internal slots on the
+        <a><code>PushSubscription</code></a>, ordered by the name of the key:
+          <ol>
+            <li>Add an entry to <var>keys</var> whose key name is the name of the key and whose
+            value is an <code>ArrayBuffer</code> containing the defined serialization of the public
+            key (for an asymmetric key pair) or the key (for a symmetric key).
+            </li>
```

Comment:
I see, thanks!

In the interest of not complicating this until we have to, we could, and probably have to define the serialization value per `PushEncryptionKeyName` entry. Since both P-256 and Curve25519 have a 32 octet binary key, this can be done regardless of our decision on the curve, and will remove this ambiguity from the initial PR. The needed complexity can then be added when additional curves get supported.